client/src/components/Export/ExportButtons/ExportButtons.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import moment from "moment";
+import ExportToCsvButton from "../ExportToCsvButton/ExportToCsvButton";
+import ExportToPdfButton from "../ExportToPdfButton/ExportToPdfButton";
+
+/**
+ * Presentational component for a button that exports data to CSV
+ */
+export default class ExportButtons extends React.Component {
+
+    render() {
+        const { fields, data, title } = this.props;
+
+        return (
+            <span>
+                <span>
+                    <ExportToCsvButton className="mt-2" fields={fields}
+                                       data={data}
+                                       exportTitle={`${title} Data ${moment().format('YYYY-MM-DD hh:mm')}`}/>
```

Comment:
Files replace the ":" with "_". This makes time part of the file name is sort of confusing to look at. I didn't know what it was supposed to be until I checked the code. Maybe remove it, I don't think the exact time is needed.

Reply:
I agree, but let's ask Dr. Decker what she thinks.  This is the format that has been in use from the previous team, so she might like it